### PR TITLE
IDE-2886 step into descriptor page when click show all page

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

**Comment by yanan** [comment]
Test Passed on Jenkins http://ide-build-1:9090/ on this commit acaa6f746027f251841b0bba0c91fc333d9c81d5 on build 103

**Comment by yanan** [comment]
Test Passed on Jenkins http://ide-build-1:9090/ on this commit c266e6472198fc12da261d297a147147b708412a on build 105

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/InitConfigureProjectPage.java [hunk 1]
```diff
@@ -1526,7 +1526,8 @@ else if( bundUrl != null && bundUrl.length() > 0 && !bundleUrlValidation.compute
 
                 if( dataModel.getImportFinished().content() )
                 {
-                    message = "import finished, please go to first page and click \"Restart...\"";
+                    message =
+                        "import finished, if you want to re-import, please go to first page and click \"Restart...\"";
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/InitConfigureProjectPage.java]: Change message to 
`"Import has finished.  If you want to reimport, please go to the first page and click "Restart..."`

Comment (AndyWu2015) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/InitConfigureProjectPage.java]: got it

Comment (AndyWu2015) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/InitConfigureProjectPage.java]: done

**Comment by yanan** [comment]
Test Passed on Jenkins http://ide-build-1:9090/ on this commit f1ff6c8c4b9cf116fcd9deb4aaaca636e72b958f on build 106

**Comment by AndyWu2015** [comment]
never mind . I will change the validation message name at a another pr.

**Pull request closed without merging**